com.playeveryware.eos/Runtime/Core/EOSManager.cs

```diff
@@ -437,28 +437,24 @@ public void RemoveManager<T>() where T : IEOSSubManager
             //-------------------------------------------------------------------------
             private Result InitializePlatformInterface()
             {
-                EOSConfig configData = Config.Get<EOSConfig>();
+                ProductConfig productConfig = Config.Get<ProductConfig>();
                 IPlatformSpecifics platformSpecifics = EOSManagerPlatformSpecificsSingleton.Instance;
 
                 print("InitializePlatformInterface: platformSpecifics.GetType() = " + platformSpecifics.GetType());
 
-                EOSInitializeOptions initOptions = new EOSInitializeOptions();
+                EOSInitializeOptions initOptions = new();
 
                 print("InitializePlatformInterface: initOptions.GetType() = " + initOptions.GetType());
 
-                initOptions.options.ProductName = configData.productName;
-                initOptions.options.ProductVersion = configData.productVersion;
+                initOptions.options.ProductName = productConfig.ProductName;
+                initOptions.options.ProductVersion = productConfig.ProductVersion.ToString();
                 initOptions.options.OverrideThreadAffinity = new InitializeThreadAffinity();
 
                 initOptions.options.AllocateMemoryFunction = IntPtr.Zero;
                 initOptions.options.ReallocateMemoryFunction = IntPtr.Zero;
                 initOptions.options.ReleaseMemoryFunction = IntPtr.Zero;
 
-                var overrideThreadAffinity = new InitializeThreadAffinity();
-
-                configData.ConfigureOverrideThreadAffinity(ref overrideThreadAffinity);
-
-                initOptions.options.OverrideThreadAffinity = overrideThreadAffinity;
+                initOptions.options.OverrideThreadAffinity = PlatformManager.GetPlatformConfig().threadAffinity.Unwrap();
```

Comment:
This should use a null check; threadAffinity can be null.

Reply:
Good catch, just fixed it.